tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -171,16 +161,15 @@
       "name": "Chemistry",
       "max_counted": 1,
       "min_needed": 1,
-      "description": "Chemistry (one semester)",
-      "explanation": null,
+      "explanation": "<b>Chemistry</b><br><i>one semester, must be completed by the end of the first year</i><hr>207 Advanced General Chemistry: Materials Chemistry, or 201 General Chemistry, or equivalent",
       "pdfs_allowed": false,
+      "double_counting_allowed": true,
```

Comment:
This and the `double_counting_allowed` on lines 119 and 201 fixes the problem where the ISC curriculum wasn't able to satisfy more than one requirement at a time.

Rather than just putting it only on the ISC subrequirement, I put it here, because otherwise, it conflicts with it being a hidden subrequirement (see https://github.com/TigerPathApp/tigerpath/issues/149). It doesn't matter because these other courses couldn't double count anywhere else anyway, so it only makes a difference for the ISC courses.